### PR TITLE
Release 0.2

### PR DESCRIPTION
**Pull request by shiroyasha**
- I've removed the release list sidebar, it is noise, reduces the importance of the main content, and I see little benefit in exploring previous versions in the sidebar. The latest release is most important.

- With that removal we don't have the list to the RSS feed. We can reintroduce it in the header or footer of the page.

- Last time we terser, this time I'm adding big glorious screenshots of the app. It is 10x more effective with conveying the message. We can keep the terser technical like message to GitHub.

- I've removed the "Operately vX.Y" from the title and moved it above the title. This reduced the noise Example left vs right:

Screenshot 2024-10-04 at 17 40 43

---

What is left:

- [x] Figure out how to move images to Astro and not use Imgur.
- [x] I need copyediting help.
- [x] I used the CTA from the last release, but we might have a better version for this one.

### CODE REVIEW

File: src/content/releases/release-v0.2.0.mdx [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+version: "0.2"
+date: 2024-10-04
+description: "TODO"
+published: true
+---
+
+Operately's latest release is packed with improvements designed to make your
+company’s operations smoother and more efficient. Here’s what’s new!
+
+## Project Retrospectives that drive results
```
Comment (shiroyasha) [inline src/content/releases/release-v0.2.0.mdx:12]: Weak and generic subtitle. It should emphasizes the benefits of doing project retros.

File: src/content/releases/release-v0.2.0.mdx [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+projects, discuss improvements openly, and learn from your team's insights.
+
+We built this feature because we know how hard it is to pause and reflect when 
+you're moving fast. But those moments of reflection are gold. They help you avoid
+repeating mistakes and double down on what's working.
+
+Try it out. Open a recent project and start a retrospective. You might be 
+surprised by the insights you uncover.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
```
Comment (shiroyasha) [inline src/content/releases/release-v0.2.0.mdx:25]: This should be wrapped in into a dedicated React component.

File: src/content/releases/release-v0.2.0.mdx [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  <img src="https://i.imgur.com/envGi8G.png" alt="Project Retrospective" className="rounded-lg border border-gray-300" />
+</div>
+
+## Onboard and Offboard people faster
+
+We've streamlined team management for company admins. The new admin page puts
```
Comment (shiroyasha) [inline src/content/releases/release-v0.2.0.mdx:31]: streamlined is a bullshit word. I need to chose a better one.

File: src/content/releases/release-v0.2.0.mdx [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+Company admins now have more control over managing and updating profiles.
+Whether it's assigning manager-reporter relationships or changing someone's
+title, it's all doable from the new admin page. Need to remove someone who has
+left the company? Their account will now be automatically suspended,
+streamlining off-boarding without the hassle.
+
+As an admin click on the company dropdown -> Company Admin -> Manage Team Members.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/ljimvRg.png" alt="People Management" className="rounded-lg border border-gray-300" />
```
Comment (shiroyasha) [inline src/content/releases/release-v0.2.0.mdx:43]: Why does the title says "Invitationss awaiting..."? Better image needed.

File: src/content/releases/release-v0.2.0.mdx [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+each responsible for a different part of the system. Now, you add all three at once, 
+assigning them specific responsibilities and keeping everyone in the loop.
+
+Why does this matter? Because in the real world, roles change. People step up.
+New folks join. You shouldn't need an IT degree to keep your project moving when 
+this happens. We built this because we've felt this pain ourselves. Unclear 
+responsibilities kill momentum. They breed confusion.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/HX85OsP.png" alt="People Management" className="rounded-lg border border-gray-300" />
```
Comment (shiroyasha) [inline src/content/releases/release-v0.2.0.mdx:63]: By the title of the segment I would expect to see detailed, project specific responsibilities on the project. Having it say "Software Developer" directly contradicting what the point of this is.

File: src/content/releases/release-v0.2.0.mdx [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+come knocking? Good luck. So we made it simple. Dead simple.
+
+We built this because we've been there. Digging through settings,
+cross-checking spreadsheets, wondering if we missed someone. It's a waste of
+time and it's nerve-wracking. Not anymore.
+
+Go to a project -> Manage Team & Access -> on the bottom click show all.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/mlQ5bZP.png" alt="See all people with access to the project" className="rounded-lg border border-gray-300" />
```
Comment (shiroyasha) [inline src/content/releases/release-v0.2.0.mdx:80]: Same problem as with the previous one.

Also, colors are off in this screenshot. The green is almost gray.

File: src/content/releases/release-v0.2.0.mdx [hunk 7]
```diff
@@ -0,0 +1,119 @@
...
+## Your Inbox, Your Rules: New Subscription Controls
+
+You can also control who will receive notifications in any type of a message
+that you send across spaces, projects, and goals.. Stay updated on the
+discussions that matter to you and tune out the ones that don’t. Whether it's
+project updates, comments, or notifications, you can easily manage what shows
+up in your inbox with just a click.
+
+<div className="drop-shadow-xl rounded-xl overflow-hidden lg:-mx-32">
+  <img src="https://i.imgur.com/vayerOI.png" alt="See all people with access to the project" className="rounded-lg border border-gray-300" />
```
Comment (shiroyasha) [inline src/content/releases/release-v0.2.0.mdx:92]: Its not clear what this screenshot is showing. It should be a gif with someone subscribing or unsubscribing from the message thread.

**Review by markoa — APPROVED** [review]
Made enough changes in the now-merged-here PR that we can proceed with publishing. Except minor edits the images have not been changed, it's good enough for the moment we're in.

**Pull request merged**